smartapps/smartthings/virtual-device-manager.src/virtual-device-manager.groovy

```diff
@@ -0,0 +1,62 @@
+/**
+ *  Virtual Device Manager
+ *
+ *  Copyright 2015 Bob Florian/SmartThings
```

Comment:
Should update copyright date/author since it's a new file

Reply:
Copyright should be the oldest based on when the file was used, right? The code is based on Bob's code written in 2015, so copyright is 2015.
older is better